draft-packages/table/KaizenDraft/Table/README.mdx

```diff
@@ -70,6 +70,9 @@ A table organizes and displays tabular data. It shows all data in a set, whereas
     *   For more than 10 rows and other data on the page, consider pagination or inline truncation & expansion.
     *   For more than 25 rows as the only table on the page, consider pagination or inline truncation & expansion.
     *   See the [pagination guide](https://cultureamp.design/components/pagination/).
+*   [Focus indicator](/guidelines/interaction-states):
+    *   On keyboard navigation select the individual interactive elements within the table.
```

Comment:
The use of bold / not bold in this list is crazy 🤪 Something for someone to clean up another time

![image](https://user-images.githubusercontent.com/315158/112239394-59313900-8c81-11eb-8e9f-d8763321371b.png)